sklearn/linear_model/tests/test_quantile.py

```diff
@@ -128,12 +129,15 @@ def test_quantile_sample_weight():
     assert weighted_fraction_below == approx(0.5, abs=3e-2)
 
 
+@pytest.mark.skipif(
+    parse_version(scipy.__version__) < parse_version("1.6.0"),
```

Comment:
Sorry, i dont understand what is your suggestion here.

Reply:
Instead of
```python
import scipy

do_something(scipy.__version__)
```
you can do
```python
from sklearn.utils.fixes import sp_version

do_something(sp_version)
```
Note that `sp_version` is already imported in this file.

Reply:
```suggestion
    sp_version < parse_version("1.6.0"),
```